mlptrain/config.py

```diff
@@ -74,6 +75,8 @@ class _ConfigClass:
         'amsgrad': True,
```

Comment:
Add patience and scheduler_patience control.
```suggestion
        'lr': 0.001,
        'patience': 50,
        'scheduler_patience': 20,
        'amsgrad': True,
```